server/src/helpers/questions.ts

```diff
@@ -134,13 +135,209 @@ export const editComment = async (
     .returning();
 };
 
-export const deleteComment = async (
-  // questionId: number,
-  answerId: number,
-  commentId: number
-) => {
+export const deleteComment = async (answerId: number, commentId: number) => {
   return await database
     .delete(comments)
     .where(and(eq(comments.id, commentId), eq(comments.answerId, answerId)))
     .returning();
 };
+
+// export const searchQuestions = async (query: string) => {
+//   if (!query.trim()) {
+//     return []; // Return an empty array for an empty query
+//   }
+
+//   return await database
+//     .select()
+//     .from(questions)
+//     .where(questions.question.toLowerCase().like(`%${query.toLowerCase()}%`));
+// };
+
+export const searchQuestions = async (query: string) => {
+  if (!query.trim()) {
+    return []; // Return an empty array for an empty query
+  }
+
+  const queryLowerCase = query.toLowerCase(); // Convert query to lowercase
+
+  return await database
+    .select()
+    .from(questions)
+    .where(questions.question.toLowerCase().like(`%${queryLowerCase}%` as any));
```

Comment:
you cannot use .toLowerCase on Drizzle like this

.toLowerCase is a JavaScript method

Please have a look at Drizzle ORM documentation and find out about transforming columns to lowercase to achieve what you want

Reply:
i have a look. i saw what you sent